aiaccel/util/process.py

```diff
@@ -128,6 +130,34 @@ def ps2joblist() -> List[dict]:
         job_list.append(d)
 
     return job_list
+'''
+
+
+def ps2joblist() -> List[dict]:
+    """Get a ps result and convert to a job list format.
+
+    Returns:
+        List[dict]: A job list of ps result.
+
+    Raises:
+        KeyError: Causes when required keys are not contained in a ps result.
+    """
+
+    job_list = []
+
+    for pInfoDict in psutil.process_iter(['pid', 'username', 'status', 'create_time', 'cmdline']):
```

Comment:
Might it be better to lowercase it to match the rest of the description?
pInfoDict -> p_info